functions/supportfiles/download-eeglab-plugin

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+#
+# Kevin Murphy
+# kevin@murp.us
+
+set -euo pipefail
+
+if [[ $# -ne 1 ]]; then
+    cat <<USAGE >&2
+usage: $0 <plugin-name>
+
+    Install a plugin called <plugin-name> from the sccn.ucsd.edu website.
+
+USAGE
+    exit 1
+fi
+
+# Go to the root of this repository so that matlab imports work correctly.
+cd "$(git rev-parse --show-toplevel)"
```

Comment:
This depends on the installation containing the actual `git` repository (not sure if that's the case for the official installer, as I just `git clone`ed this onto my system).